mock-http/main.bal

```diff
@@ -5,13 +5,33 @@ import ballerina/file;
 configurable string path = ?;
 configurable int case = ?;
 
+type S3Response record {|
+  string x\-amz\-id\-2;
+  string ETag;
+  byte[] data;
+|};
+
 service / on new http:Listener(80) {
 
-  resource function default [string... parm](http:Request req) returns anydata|error {
+  resource function default [string... parm](http:Request req) returns anydata|http:Response|error {
     string host = check req.getHeader("host");
     string fileName = string `${path}/case-${pad2(case)}-${host}`;
     if check file:test(fileName + ".json", file:EXISTS) {
-      return io:fileReadJson(fileName + ".json");
+      S3Response jsonData = check (check io:fileReadJson(fileName + ".json")).cloneWithType();
+      
+      if fileName.includes("s3") {
```

Comment:
Deciding this on the fileName is not correct. Because host can contain `s3` which cannot restrict. What we should do is we should decide using the json we read.

Reply:
Updated [21aad91](https://github.com/ballerina-guides/enterprise-integration-patterns/pull/24/commits/21aad9174100dbcf3ef615c3f08ce295be8b6c83)